examples/MNIST/MLproject

```diff
@@ -0,0 +1,18 @@
+name: mnist-example1
+
+conda_env: conda.yaml
+
+entry_points:
+  main:
+    parameters:
+      max_epochs: {type: int, default: 5}
+      model_save_path: {type: str, default: 'model'}
```

Comment:
@karthik-77 parameters might need to be synced with  the command  from [repository](https://github.com/mlflow/mlflow/tree/master/examples/pytorch/MNIST/example1#passing-custom-training-parameters)

Reply:
We decided not to have to many arguments,since this is an deployment example. The parameters are in sync with [mnist_model.py](https://github.com/mlflow/mlflow-torchserve/blob/master/examples/MNIST/mnist_model.py)